recipes/orthanq/meta.yaml

```diff
@@ -5,7 +5,8 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
+  skip: True  # [osx]
```

Comment:
What is the reason for this skip ?
Neither the title or description of the PR explains any problems with OSX,

Reply:
The updated recipe now requires [vg](https://bioconda.github.io/recipes/vg/README.html#package-package%20&#x27;vg&#x27;). That package is not available for osx.